linked_list/LinkedList.c

```diff
@@ -0,0 +1,176 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node {
+	int d;		//	integer type data  
+	struct Node* next;		//	pointer to same structure
+}*start;	//	first node of the list
+
+int main() {
+	start=NULL;
+	// Adding element to the list
+	addFront(23);
+	traverse();
+	
+	// Adding elements at the end
+	addLast(25);
+	addLast(46);
+	traverse();
+	
+	// Adding elements to front
+	addFront(12);
+	addFront(11);
+	traverse();
+	
+	// Adding element at a given index (Adding 4 at index 3)
+	addAtPos(14, 3);
+	traverse();
+	
+	// Removing First Element
+	deleteFirst();
+	traverse();
+	
+	// search
+	search(14);
+	search(12);
+	
+	// Removing Last Element
+	deleteLast();
+	traverse();
+	
+	// Removing an element from a index
+	deleteAtpos(17);
+	traverse();
+	return 0; 
+}
+
+//  Adding at Front
+void addFront(int data) {
```

Comment:
try to give definition of all the functions at top. Else give prototyping of each function. Also all other C/C++ Codes have function declaration's at top, so try to follow the convention.  And also place some comment or doc string stating about the parameters and return type.